### ISSUE TITLE
WSL no internet connection

### ISSUE THREAD

**Issue by spyrosavl** [microsoft/vscode#154523]
Issue Type: <b>Bug</b>

Since I updated to the latest version, I do not have access to the internet from WSL. For example, "wget google.com" fails with "wget: unable to resolve host address ‘google.com’".

VS Code version: Code 1.69.0 (92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9, 2022-07-07T05:28:36.503Z)
OS version: Windows_NT x64 10.0.19044
Restricted Mode: No
Remote OS version: Linux x64 5.10.102.1-microsoft-standard-WSL2

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|11th Gen Intel(R) Core(TM) i5-1145G7 @ 2.60GHz (8 x 1498)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.73GB (6.67GB free)|
|Process Argv|--crash-reporter-id 75a5c44a-0982-43c6-bcbd-63efaa860566|
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: Ubuntu|
|OS|Linux x64 5.10.102.1-microsoft-standard-WSL2|
|CPUs|11th Gen Intel(R) Core(TM) i5-1145G7 @ 2.60GHz (8 x 1497)|
|Memory (System)|12.26GB (10.98GB free)|
|VM|0%|
</details><details><summary>Extensions (16)</summary>

Extension|Author (truncated)|Version
---|---|---
jupyter-keymap|ms-|1.0.0
remote-ssh|ms-|0.82.1
remote-ssh-edit|ms-|0.80.0
remote-wsl|ms-|0.66.3
vscode-markdownlint|Dav|0.47.0
gitlens|eam|12.1.1
copilot|Git|1.31.6194
gc-excelviewer|Gra|4.2.55
python|ms-|2022.10.0
vscode-pylance|ms-|2022.7.20
jupyter|ms-|2022.6.1001902341
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.8
makefile-tools|ms-|0.5.0
code-spell-checker|str|2.3.0
html-preview-vscode|tht|0.2.5

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627:30244334
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscorecescf:30445987
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
vsc1dst:30438360
pythonvs932:30410667
wslgetstarted:30449410
vscscmwlcmt:30465135
cppdebug:30492333
vscaat:30438848
vsclangdc:30486549

```

</details>

**Comment by spyrosavl** [comment]
Yes, it happens outside of VS Code too. I restarted WSL but it did not fix it.

**Comment by aeschli** [comment]
Maybe PROXY settings?

I'm afraid you'll have to file an issue against wsl: https://github.com/microsoft/wsl
